### ISSUE TITLE
Feature Request: Implement group messages/unacknowledged messages to use APRS services

### ISSUE THREAD

**Issue by fkaup** [ge0rg/aprsdroid#339]
Implementing the option for messages to not ask for acknowledgment (i.e. append {nnnn to each message) allows to use APRS group messages or services like group messages, ping, call sign look-up, or (self-)spotting (basic services defined on [aprs-is.net](http://aprs-is.net/APRSServices.aspx), [SOTA2APRS](https://www.sotaspots.co.uk/Aprs2Sota_Info.php), or [APSPOT](https://apspot.radio/)).

These services are partially not configured to send proper ACKs but instead return useful information in a separate message. The current APRSdroid implementation repeats each such message 6 times, needlessly increasing network load.

**Comment by ge0rg** [comment]
Yeah, this is quite a pain point. I really need to add the respective UI to the app. Right now, the workaround is to long-press and cancel a message after it was answered by the automatic station.
